src/main/java/org/prebid/server/spring/config/ServiceConfiguration.java

```diff
@@ -79,6 +79,7 @@ CacheService cacheService(
             @Value("${cache.query}") String query,
             @Value("${cache.banner-ttl-seconds:#{null}}") Integer bannerCacheTtl,
             @Value("${cache.video-ttl-seconds:#{null}}") Integer videoCacheTtl,
+            @Value("${events.urltemplate}") String eventsUrlTemplate,
```

Comment:
Consider to singular for config prefix - `event.url-template`. It is because the property relates to our `/event` endpoint.

Reply:
nvm, leave `events.*` so far.

Reply:
Pls see updated config -- https://github.com/prebid/prebid-server/issues/1015